test/e2e-common.sh

```diff
@@ -288,6 +288,22 @@ function install() {
     > "${ytt_result}" \
     || fail_test "failed to create deployment configuration"
 
+  # use emptyFieldMatcher to avoid aggregation rule conflict.
+  # see: https://github.com/vmware-tanzu/carvel-kapp/issues/227
+  cat <<EOF >> ${ytt_result}
+---
+apiVersion: kapp.k14s.io/v1alpha1
+kind: Config
+rebaseRules:
+- path: [rules]
+  type: copy
+  sources: [existing, new]
+  resourceMatchers:
+  - notMatcher:
+      matcher:
+        emptyFieldMatcher:
+          path: [aggregationRule]
+EOF
```

Comment:
Can we pull this into a config file - ie. under `test/config/ytt/core` maybe?

Reply:
Sure! Updated.